rust-container/Makefile

```diff
@@ -0,0 +1,34 @@
+all:	rust-container
+
+rust-container:
+	git clone https://github.com/sclorg/rust-container.git
+	cd rust-container ; \
+	git submodule update --init ; \
+	patch -p 1 -i ../rust-container-s390x.patch ; \
+	make
+
+push:	
+	if [ -d rust-container ]; then \
+		../pushit.sh httpd ; \
+	fi
+	@rm -rf rust-container
+
+tag:	
+	if [ -d rust-container ]; then \
+		cd rust-container ; \
+		make tag ; \
+		../../retag.sh httpd ; \
```

Comment:
Correct me if I am wrong but this looks like a copy paste error. I am assuming that you copied this Makefile from httpd directory and forgot to change it to rust. Same error in Line 12

Reply:
Yes, my bad. let me fix it